tests/lib/generators.py

```diff
@@ -0,0 +1,116 @@
+"""
+The functions from this file will be useful once CAAS-1623 has been implemented
+"""
+import os
+from .constants import *
+
```

Comment:
Same here, remove methods that will become redundant once you clean the errors above.